nbflow/doit-example/README.md

```diff
@@ -0,0 +1,7 @@
+# Example nbflow setup
```

Comment:
I would prefer if we actually kept just a single example that can be run multiple ways -- so if we have the `SConstruct` and `dodo.py` side-by-side and users can either run `scons` or `doit` depending on which system they want to try. Then we also don't need to maintain multiple almost identical examples.